modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/build.gradle

```diff
@@ -0,0 +1 @@
+dependencies {}
```

Comment:
I never saw this file empty before. It's intended to be empty?

Reply:
It's emptry because we haven't gradle dependencies at this time and, i guess, if ths file is present we should be able to start it as a module